### ISSUE TITLE
How does DigitsDG divide the training set and test set? The division method is not given in the data_list folder?

### ISSUE THREAD

**Issue by helloworldzsq** [zzwdx1234/MEDIC#5]

**Comment by zzwdx** [comment]
You can download from this link [url](https://drive.google.com/file/d/1hjxzeF8pJSU4WcPBDDdKd86IZRJw1Rpf/view?usp=sharing).
